.rubocop.yml

```diff
@@ -1,5 +1,6 @@
 AllCops:
   NewCops: enable
+  TargetRubyVersion: 2.5
```

Comment:
To fix Rubocop

![Screen Shot 2565-01-18 at 16 54 48](https://user-images.githubusercontent.com/6965195/149914048-b7fef9a9-f6ac-4326-9464-cfcdfe360024.png)